src/LibraryView.tsx

```diff
@@ -17,13 +18,16 @@ export class LibraryView {
     htmlElementId: string = "";
     loadedTypesJson: any = null;
     layoutSpecsJson: any = null;
+    reactor: Reactor = null;
 
     constructor(config: LibraryViewConfig) {
 
         this.setLoadedTypesJson = this.setLoadedTypesJson.bind(this);
         this.setLayoutSpecsJson = this.setLayoutSpecsJson.bind(this);
         this.prefetchContents = this.prefetchContents.bind(this);
         this.updateContentsInternal = this.updateContentsInternal.bind(this);
+        LibraryView.prototype.reactor = new Reactor();
```

Comment:
Since we're in TypeScript, let's not use `prototype` anymore. It can just be `this.reactor = new Reactor();`.